exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,4 +1,7 @@
 // Declare your function here
+function createLongGreeting(name, age){
```

Comment:
Understanding string concaternation takes basic exercises like these. You did well in this exercise analysing your code and code stucture.